docs/faq/autoscaling.md

```diff
@@ -0,0 +1,51 @@
+
+# What is autoscaling?
+
+Autoscaling refers to a system where the resources allocated to an application
+can be dynamically adjusted in response to load on the system.
+
+In Fleet, autoscaling is applied to application server instances (Frontend
+instances). This is designed to allow an application to scale across a
+reasonable range of traffic levels. There will eventually be scalability
+limits in regards to the backend (database) for a given size, so the Fleet size
+should be chosen appropriately such that the database can serve the range of
+loads expected.
+
+Autoscaling in Fleet is designed to cope with the normal changes in traffic over a day, but is not
```

Comment:
Consider removing 'in Fleet'.  This is a caveat for all autoscaling set ups; not just Fleet's.